### ISSUE TITLE
TouchableHighlight or TouchableOpacity doesn't work inside Callout

### ISSUE THREAD

**Issue by jojonarte** [lelandrichardson/react-native-maps#226]
This is my current code setup in render

```
<View style={styles.view_container}>

              <MapView
                    style={styles.map}
                    region={this.state.region}
                    showsUserLocation={YEAH}
                    followsUserLocation={YEAH}
                    onRegionChange={this.onRegionChange}
                  >
                    {this.state.markers.map(marker => (
                      <MapView.Marker
                        coordinate={marker.latlng}
                      >

                         <MapView.Callout>
                            <CustomCallout {...marker}/>

                         </MapView.Callout>

                        </MapView.Marker>
                    ))}
              </MapView>
              <TextInput />

            </View>
```

I can't seem to get the custom callout to listen to presses done inside the CustomCallout. The code is this.

```
class CustomCallout extends Component{
  paypal(){
    console.log('presseeedd')
    }

  componentDidMount(){
    console.log('custom call props', this.props.title);
  }

  render(){
        return (
          <View style={{flex:1, flexDirection:'column'}}>
              <Text>{this.props.title}</Text>
              <Text>{this.props.description}</Text>
              <TouchableHighlight onPress={this.paypal.bind(this)}>
                <Text>Reserve Now</Text>
              </TouchableHighlight>
          </View>
        )
  }
}
```

**Comment by jrichardlai** [comment]
Is this happening for Android and iOS?

**Comment by jojonarte** [comment]
It's happening on Android.

**Comment by jrichardlai** [comment]
Seems to be linked https://github.com/lelandrichardson/react-native-maps/issues/211

**Comment by heydabop** [comment]
Android map callouts don't support buttons inside of them, only an onPress event for the entire callout. This is a limitation of the Google Maps library on Android and react-native-maps can't do anything about it.

**Comment by jrichardlai** [comment]
Closing this one too.

**Comment by eugenehp** [comment]
@jojonarte hey Jojo, were you able to solve it?

**Comment by heydabop** [comment]
@eugenehp Read my previous comment.

**Comment by eugenehp** [comment]
Ross, thanks. Let me rephrase it, have you found any solution on top of the
maps that allowed you to do your initial intend? Basically I'm asking what
you end up using in it. Thanks in advance!

**Comment by jojonarte** [comment]
@eugenehp I wasn't able to, what I did to use it was just let the callout be tapped to view another scene with the full details of what's supposedly pointed by the callout.

**Comment by eugenehp** [comment]
Thanks Jojo, that makes sense. Thanks for your time! 👍

**Comment by jojonarte** [comment]
Glad I could help.

**Comment by ddave09** [comment]
@jojonarte would you care to share an example of that? I'm trying to do something similar.

**Comment by muhammadqazi** [comment]
you can use CalloutSubView for this purpose

```
<Callout style={styles.calloutWidth}>
     <CalloutSubview
           style={styles.phoneContainer}
          onPress={() => {
       console.log('onPress Clicked')
      }}>


    <Icon
         name="directions-car"
         type="material"
         color={APP_THEME.APP_ORANGE}
         size={30}
          underlayColor="clear"
     />


       </CalloutSubview>
</Callout>
```

**Comment by jerom98** [comment]
> you can use CalloutSubView for this purpose
> 
> ```
> <Callout style={styles.calloutWidth}>
>      <CalloutSubview
>            style={styles.phoneContainer}
>           onPress={() => {
>        console.log('onPress Clicked')
>       }}>
> 
> 
>     <Icon
>          name="directions-car"
>          type="material"
>          color={APP_THEME.APP_ORANGE}
>          size={30}
>           underlayColor="clear"
>      />
> 
> 
>        </CalloutSubview>
> </Callout>
> ```

CalloutSubview works only on IOS

**Comment by Elidor00** [comment]
@heydabop 

Is your answer still valid after 8 years or is there now a solution for Android too?
